ert_shared/exporter.py

```diff
@@ -29,32 +30,24 @@ def is_valid(self):
         runpath_job = ERT.enkf_facade.get_workflow_job(self._runpath_job)
 
         if export_job is None:
-            error = "Export not available due to {job_name} is not installed.".format(
-                job_name=self._export_job
-            )
-            logging.warning(error)
+            error = f"Export not available due to {self._export_job} is not installed."
+            logger.warning(error)
             return False
 
         if runpath_job is None:
-            error = "Export not available due to {job_name} is not installed.".format(
-                job_name=self._runpath_job
-            )
-            logging.warning(error)
+            error = f"Export not available due to {self._runpath_job} is not installed."
+            logger.warning(error)
```

Comment:
`logger.warning(error)` sounds a bit strange, the variable name suggests it should be an `error`?

Reply:
In this case an absence of the runpath_job would be an error, as that should not happen, so something has gone horribly wrong, best make it an `logger.error`, though the whole construction of this class is a bit baffling to me (but that is besides this PR)